app/src/main/java/com/pileproject/drive/setting/app/ProgramListFragment.java

```diff
@@ -80,17 +92,25 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         mProgramsCheckAllButton[0] = (Button) v.findViewById(R.id.programList_samplePrograms_checkAllButton);
         mProgramsCheckAllButton[1] = (Button) v.findViewById(R.id.programList_userPrograms_checkAllButton);
 
+
         return v;
     }
 
     @Override
     public void onActivityCreated(Bundle savedInstanceState) {
         super.onActivityCreated(savedInstanceState);
 
+        final ProgramListFragment thiz = this;
```

Comment:
Oh, I was confused... I have no good idea now;( what should we do in normal (?) way? Something like this: http://stackoverflow.com/questions/22777928/button-click-inside-fragment-to-open-up-new-fragment ?